### PR TITLE
London Class 8 - Irina Shilova - JavaScript Core 2 - WEEK 1 

### PR DESCRIPTION
**Pull request by IrinShilova**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

**Comment by IrinShilova** [comment]
Hello @moneyinthesky, it's done

File: 2-mandatory/1-recipes.js [hunk 1]
```diff
@@ -22,4 +22,42 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+
```
Comment (moneyinthesky) [inline 2-mandatory/1-recipes.js:26]: These look good!

File: 2-mandatory/2-currency-code-lookup.js [hunk 2]
```diff
@@ -18,7 +18,11 @@ const COUNTRY_CURRENCY_CODES = [
 ];
 
 function createLookup(countryCurrencyCodes) {
-  // write code here
+  let dict = {};
```
Comment (moneyinthesky) [inline 2-mandatory/2-currency-code-lookup.js:21]: Well done :)

File: 2-mandatory/3-shopping-list.js [hunk 3]
```diff
@@ -19,7 +19,11 @@ let pantry = {
 };
 
 function createShoppingList(recipe) {
```
Comment (moneyinthesky) [inline 2-mandatory/3-shopping-list.js:21]: This looks perfect!

**Comment by moneyinthesky** [comment]
Hey @IrinShilova, very good job on this coursework! I didn't have too much to add :)
Maybe if you have time, you can come back and attempt the extra exercise as well - it will be good practice!

**Pull request closed without merging**